src/components/global-view-core-row.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React from 'react';
+import { Button, Grid, TableCell, TableRow } from '@material-ui/core';
+import { gridcapaFormatDate, sha256 } from './commons';
+import { ListAlt, Visibility } from '@material-ui/icons';
+import { TaskStatusChip } from './task-status-chip';
+import { RunButton } from './run-button';
+import FileSummary from './file-summary';
+
+const GlobalViewCoreRow = ({
+    step,
+    index,
+    page,
+    rowsPerPage,
+    handleFileOpen,
+    handleEventOpen,
+}) => {
+    let timestamp = step.timestamp;
+    let formattedTimestamp = gridcapaFormatDate(timestamp);
+    let encryptedMessage = sha256(formattedTimestamp);
+
+    return (
+        step.taskData && (
+            <TableRow hover key={encryptedMessage}>
+                <TableCell
+                    data-test={encryptedMessage + '-process-event-timestamp'}
+                    size="small"
+                >
+                    {formattedTimestamp}
+                </TableCell>
+                <TableCell size="small">
+                    <Grid container direction="row" spacing={2}>
+                        <Grid item>
+                            <Grid container direction="column">
+                                <Grid item>
+                                    <FileSummary
+                                        type="Input"
+                                        listoffile={step.taskData.inputs}
+                                    />
+                                </Grid>
+                                <Grid item>
+                                    <FileSummary
+                                        type="Output"
+                                        listoffile={step.taskData.outputs}
```

Comment:
can you rename this variable "listoffile " to respect the variable naming conventions : (start with a lowercase letter and then capitalize the first letter of every subsequent word)

Reply:
renamed to listOfFile